app/controllers/coronavirus_form/know_nhs_number_controller.rb

```diff
@@ -31,6 +31,6 @@ def submit
 private
 
   def previous_path
-    medical_conditions_path
+    name_path
```

Comment:
I think this should be: `contact_details_path`

Reply:
Good spot.  We should probably have a feature test for the back links.  It has caught us out (on the business volunteering form) a few times.